test/test_benchmark.py

```diff
@@ -305,52 +333,58 @@ def get_output_files(self):
 
 benchmarkers: List[Benchmarker] = []
 
-if not common.EMTEST_FORCE64:
-  benchmarkers += [
-    NativeBenchmarker('clang', [CLANG_CC], [CLANG_CXX]),
-    # NativeBenchmarker('gcc',   ['gcc', '-no-pie'],  ['g++', '-no-pie'])
-  ]
-
-if config.V8_ENGINE and config.V8_ENGINE in config.JS_ENGINES:
-  # avoid the baseline compiler running, because it adds a lot of noise
-  # (the nondeterministic time it takes to get to the full compiler ends up
-  # mattering as much as the actual benchmark)
-  aot_v8 = config.V8_ENGINE + ['--no-liftoff']
-  default_v8_name = os.environ.get('EMBENCH_NAME') or 'v8'
-  if common.EMTEST_FORCE64:
-    benchmarkers += [
-      EmscriptenBenchmarker(default_v8_name, aot_v8, ['-sMEMORY64=2']),
-    ]
-  else:
-    benchmarkers += [
-      EmscriptenBenchmarker(default_v8_name, aot_v8),
-      EmscriptenBenchmarker(default_v8_name + '-lto', aot_v8, ['-flto']),
-      EmscriptenBenchmarker(default_v8_name + '-ctors', aot_v8, ['-sEVAL_CTORS']),
-    ]
-  if os.path.exists(CHEERP_BIN):
-    benchmarkers += [
-      # CheerpBenchmarker('cheerp-v8-wasm', aot_v8),
-    ]
-
-if config.SPIDERMONKEY_ENGINE and config.SPIDERMONKEY_ENGINE in config.JS_ENGINES:
-  # TODO: ensure no baseline compiler is used, see v8
+if common.EMTEST_BENCHMARK_SIZE_FILE:
+  TEST_REPS = 0
   benchmarkers += [
-    # EmscriptenBenchmarker('sm', SPIDERMONKEY_ENGINE),
+    EmscriptenBenchmarker('size', None, []),
```

Comment:
I'm not sure we want to keep adding environment variables as a way to configure which benchmakers we run.    I had the same objection when `EMTEST_FORCE64` was added I think.
  
  We should probably come up with some kind of command line scheme for configuring the benchmarkers.   I think most ppl just tend to edit this file as they need before the run the tests but that doesn't seem great either.

Reply:
Perhaps we could give each of these benchmarkers a unique string name and then have something like `EMTEST_BENCHMARKERS=...`  ?